### ISSUE TITLE
NodeRED crashes/restarts when node is unable to log-in to servers (Skoda servers down)

### ISSUE THREAD

**Issue by Kositch** [MartinGris/node-red-contrib-skoda-connect#17]
When Skoda connect servers are unavailable (very often) nodes skoda-get or skoda-set cause whole NodeRED server to restart when not able to log-in. Can this please be solved somehow? It works when servers are available, but when they are for example for 1 hour down, it causes constant NodeRED restarts.

**Comment by foorschtbar** [comment]
Same problem here.

**Comment by Kositch** [comment]
Do you have some temporary solution so I dont have to disable node each time servers are not available? Thanks!

**Comment by foorschtbar** [comment]
Yes. I use ioBroker with the vw-connect adapter and publish the changes via MQTT back to Node-RED. The ioBroker adapter is well maintained.

**Comment by Kositch** [comment]
> Yes. I use ioBroker with the vw-connect adapter and publish the changes via MQTT back to Node-RED. The ioBroker adapter is well maintained.

I am not familiar with ioBroker - it needs some new server running on RPi?

**Comment by foorschtbar** [comment]
I use Docker on my RPi and ioBroker is just another container

**Comment by Kositch** [comment]
> I use Docker on my RPi and ioBroker is just another container

Yeah, that is sad, that this plugin is not maintained, I would pay a beer for solution of this issue :-)

**Comment by foorschtbar** [comment]
i simple try-catch could help, but i had trouble to find the position where the exception happens.

**Comment by Kositch** [comment]
> i simple try-catch could help, but i had trouble to find the position where the exception happens.

I am not so much experienced in that, if someone would fork this plugin a resolves this single issue, I would be grateful :-)
